chaco/plots/lineplot.py

```diff
@@ -122,7 +122,7 @@ def hittest(self, screen_pt, threshold=7.0, return_distance=False):
             # screen_pt is one of the points in the lineplot
             data_pt = (self.index.get_data()[ndx], self.value.get_data()[ndx])
             if return_distance:
-                scrn_pt = self.map_screen(data_pt)
+                scrn_pt = self.map_screen(data_pt)[0]
```

Comment:
Note this change is also made in #802 

Both the changes to `BaseXYPlot1.map_screen` done there and `LinearMapper.map_screen` here can cause the return type to be an an array of shape (N,2).